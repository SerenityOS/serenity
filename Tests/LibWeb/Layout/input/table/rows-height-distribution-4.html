<style>
* {
    font-family: 'SerenitySans';
}

.table {
    display: table;
    width: 200px;
    height: 300px;
    background-color: darkgoldenrod;
}

.row {
    display: table-row;
}

.cell {
    display: table-cell;
}

.a {
    background-color: darkcyan;
    height: 50px;
}

.b {
    background-color: orangered;
    height: 50%;
}
</style>
<div class="table"><div class="row a"><div class="cell">a</div></div><div class="row b"><div class="cell">b</div></div></div>