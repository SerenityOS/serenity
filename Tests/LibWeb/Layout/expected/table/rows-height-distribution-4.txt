Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x300 children: not-inline
      TableWrapper <(anonymous)> at (8,8) content-size 200x300 [BFC] children: not-inline
        Box <div.table> at (8,8) content-size 200x300 table-box [TFC] children: not-inline
          Box <div.row.a> at (8,8) content-size 200x100 table-row children: not-inline
            BlockContainer <div.cell> at (8,8) content-size 200x17 table-cell [BFC] children: inline
              frag 0 from TextNode start: 0, length: 1, rect: [8,8 9.34375x17] baseline: 13.296875
                  "a"
              TextNode <#text>
          Box <div.row.b> at (8,108) content-size 200x200 table-row children: not-inline
            BlockContainer <div.cell> at (8,108) content-size 200x17 table-cell [BFC] children: inline
              frag 0 from TextNode start: 0, length: 1, rect: [8,108 9.46875x17] baseline: 13.296875
                  "b"
              TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x600]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x300]
      PaintableWithLines (TableWrapper(anonymous)) [8,8 200x300]
        PaintableBox (Box<DIV>.table) [8,8 200x300]
          PaintableBox (Box<DIV>.row.a) [8,8 200x100]
            PaintableWithLines (BlockContainer<DIV>.cell) [8,8 200x100]
              TextPaintable (TextNode<#text>)
          PaintableBox (Box<DIV>.row.b) [8,108 200x200]
            PaintableWithLines (BlockContainer<DIV>.cell) [8,108 200x200]
              TextPaintable (TextNode<#text>)
